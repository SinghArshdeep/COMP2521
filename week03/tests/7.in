50

23


41


53

23
45
79
68


89

