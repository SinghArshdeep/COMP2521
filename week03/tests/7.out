Tree:
   50
   / \
  /   \
 /     \
23     53
 \       \
 41      79
   \     / \
   45   /   \
       68   89

#nodes = 8,  #leaves = 3
Infix  : 23 41 45 50 53 68 79 89 
Prefix : 50 23 41 45 53 79 68 89 
Postfix: 45 41 23 68 89 79 53 50 
ByLevel: 50 23 53 41 79 45 68 89 
